CorsixTH/Lua/entities/humanoid.lua

```diff
@@ -341,6 +345,18 @@ function Humanoid:afterLoad(old, new)
   if old < 134 and new >= 134 then
     self.staff_change_callbacks = {}
   end
+  if old < 210 then
+    -- We renamed the old sad7 to sad2; and sad2 to dying1. Make sure we adjust
+    -- sad2 to the new mood
+    if self:isMoodActive("sad2") then
+      self:setMood("sad2", "deactivate")
+      self:setMood("dying1", "activate")
+    end
+    if self:isMoodActive("sad7") then
+      self:setMood("sad7", "deactivate")
+      self:setMood("sad2", "activateW")
+    end
```

Comment:
This is fine. An alternative can be

``` lua
local changed_moods = {{old="sad2", new="dying1"}, {old="sad7", new="sad2"}}
for _, change in ipairs(changed_moods) do
  if self:isMoodActive(change.old) then
    self:setMood(change.old, "deactivate")
    self:setMood(change.new, "activate")
  end
end
```

Reply:
I somewhat wonder if the afterload is worth it, assuming most moods are temporary, but this is fine.